### ISSUE TITLE
Changing files in the mock folder causes all HTML files to reload

### ISSUE THREAD

**Issue by jhadenfeldt** [virtualidentityag/gulp-frontend-boilerplate#25]
When changing a JSON file within the mock folder, livereload is being triggered for all HTML files of the project which can take a while for larger projects.

**Comment by janrembold** [comment]
That happens because you never know which file is affected. So I don't think this is a bug.

**Comment by jhadenfeldt** [comment]
But is it really necessary to watch for the JSON files within the [zetzer watch task](https://github.com/virtualidentityag/gulp-frontend-boilerplate/blob/master/gulp/tasks/zetzer.js#L92)? I can't think of a situation where this is needed.

**Comment by janrembold** [comment]
This is mostly necessary for zetzers' renderHbs() helper. We use that function more and more.
It's no problem to disable that watch task per project if no renderHbs is used, but I don't like the idea to disable it by default.

**Comment by timomayer** [comment]
> It's no problem to disable that watch task per project 
no. this cant be the solution because we dont want to change gulpfile and files in gulp folder per project to stay updateable

**Comment by timomayer** [comment]
@janrembold how about to use a specific js file for handlebars helpers and ignore the other files in mock folder?

**Comment by janrembold** [comment]
> how about to use a specific js file for handlebars helpers and ignore the other files in mock folder?

That is no hbs helper issue. This is related to the watch:json task. 

But I agree with not changing the gulp files/folders. I already started a branch for an optional /boilerplate.json configuration file that overrides the default configuration. With this kind of configuration we could manage things like json watching.
